modelcontextprotocol/server.py

```diff
@@ -342,7 +343,19 @@ def update_assets_tool(
             attribute_values=["VERIFIED"]
         )
 
-        # Update user description for multiple assets
+        # Update readme for a single asset with HTML
+        result = update_assets_tool(
+            assets={
+                "guid": "asset-guid-here",
+                "name": "Asset Name",
+                "type_name": "Asset Type Name",
+                "qualified_name": "Asset Qualified Name"
+            },
+            attribute_name="readme",
+            attribute_values=["<h1>New readme content</h1>"]
```

Comment:
We should remove html based readmes from examples

Reply:
Resolved via : [d928d40](https://github.com/atlanhq/agent-toolkit/pull/63/commits/d928d40e64848f3a581d4fab5f8e94422a884ccb)